Composer/packages/client/src/pages/botProject/styles.ts

```diff
@@ -126,7 +132,7 @@ export const unknownIconStyle = (required: boolean) => {
   };
 };
 
-export const columnSizes = ['300px', '150px', '150px'];
+export const columnSizes = ['300px', '150px', '300px'];
```

Comment:
Should we just reuse the first column size instead of having two '300px'?

Reply:
I have instead changed the reference for just the table that needs 300px as column three